### PR TITLE
[READY] Makes us dreamchecker compatibile

### PR DESCRIPTION
**Pull request by AffectedArc07**
## About The Pull Request
This makes our code compatible with DreamChecker, a linter which checks BYOND code for edge cases which can break during execution. My end goal is to integrate this into travis, so I am cleaning the code for now.  This PR **WILL** need a TM as a lot of stuff has changed, but none of the behaviour of stuff should have changed, so we need to be careful of edge-cases

## Why It's Good For The Game
Code that avoids edge-cases is good code

## Changelog
:cl: AffectedArc07
code: Fixes a LOT of code edge cases
/:cl:

### CODE REVIEW

File: code/game/objects/items/implants/implantchair.dm [hunk 1]
```diff
@@ -190,4 +190,4 @@
 	brainwash(C, objective)
 	message_admins("[ADMIN_LOOKUPFLW(user)] brainwashed [key_name_admin(C)] with objective '[objective]'.")
 	log_game("[key_name(user)] brainwashed [key_name(C)] with objective '[objective]'.")
-	return TRUE
\ No newline at end of file
+	return TRUE
```
Comment (CornMyCob) [inline code/game/objects/items/implants/implantchair.dm]: honk?

**Comment by CornMyCob** [comment]
After one glance over I don't immediately see anything fucked about it. I'll go over it again tomorrow and after we testmerge it.

File: code/game/objects/items/implants/implantchair.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/objects/items/implants/implantchair.dm]: `editorconfig` forces new lines on files you touch. This was one that I opened and saved out of habit.

**Comment by AffectedArc07** [comment]
2 rounds with no glaring issues, if it breaks theres always the revert button

**Pull request merged**